pom.xml

```diff
@@ -1530,8 +1530,7 @@
     <protobuf.plugin.version>0.6.1</protobuf.plugin.version>
     <thrift.path>thrift</thrift.path>
     <thrift.version>0.12.0</thrift.version>
-    <zookeeper.version>3.4.14</zookeeper.version>
-    <!-- What ZooKeeper 3.4.x depends on and nothing more -->
+    <zookeeper.version>3.5.7</zookeeper.version>
```

Comment:
should we also update the jline version? I think it was added because of zookeeper, see https://issues.apache.org/jira/browse/HBASE-20108

zookeeper 3.5.7 is using jline 2.11

however, if we increment the JLine version, then the assembly:single maven target will fail, as the license part in JLine pom changed a bit (they named it 'The BSD License' instead of 'BSD License'). To make it work, we need to add a new supplement to supplemental-models.xml like:

```
  <supplement>
    <project>
      <groupId>jline</groupId>
      <artifactId>jline</artifactId>
      <name>JLine</name>
      <version>2.11</version>
      <licenses>
        <license>
          <name>BSD License</name>
          <url>http://www.opensource.org/licenses/bsd-license.php</url>
          <distribution>repo</distribution>
        </license>
      </licenses>
    </project>
  </supplement>
```